fulp_modules/features/jobs/brigdoc/brigdoc_clothing.dm

```diff
@@ -17,6 +17,7 @@ Contains:
 	worn_icon = 'fulp_modules/features/jobs/icons/clothing_worn.dmi'
 	icon = 'fulp_modules/features/jobs/icons/clothing_icons.dmi'
 	icon_state = "beret_medbay"
+	armor = list(MELEE = 20, BULLET = 60, LASER = 10, ENERGY = 10, BOMB = 30, BIO = 10, FIRE = 50, ACID = 60, WOUND = 5)
```

Comment:
why the fuck does it have 60 bullet armor its a fucking brig eoc berrt what the fick

Reply:
please make it more bio-resistant but nerf the armor values WHAT THE FUCK

Reply:
THESE ARE LITERALLY THE ARMOR VALUES IT INITIALLY INHERITED PUCCI DID YOU NOT KNOW????

Reply:
NO I DIDNT

Reply:
Add a oneline suggestion to your comment and I'll commit it even if I do hate your fucking guts

Reply:
JOHN WILLARD DID THIS HE MADE THE BERET A FULP BERET SUBTYPE

Reply:
BLAME @JohnFulpWillard FUCK YOU FICK YOU

Reply:
WTF DO YOU MEAN

Reply:
I DIDNT DO IT AND EVEN IF I DID I WOULDVE DONE IT BECAUSE REGULAR BERETS WERE MADE GAGS

Reply:
THATS EXACTLY WHY YOU DID IT YOU FUCKJG IDITO AJFOSMVIEEJ
YOU MADE IT THAT WAY BECAUSE OF GAGS BERETS FUCK YOU @GODLENAPHRAHREX